requirements.txt

```diff
@@ -1,3 +1,4 @@
+wcwidth==0.2.13
```

Comment:
is this needed?

Reply:
yes or else it won't pass the dependencies test. It broke earlier